crates/assistant/src/assistant_settings.rs

```diff
@@ -19,7 +19,7 @@ impl OpenAiModel {
         match self {
             OpenAiModel::ThreePointFiveTurbo => "gpt-3.5-turbo-0613",
             OpenAiModel::Four => "gpt-4-0613",
-            OpenAiModel::FourTurbo => "gpt-4-1106-preview",
+            OpenAiModel::FourTurbo => "gpt-4-0125-preview",
```

Comment:
Hardcoding this will mean having to update this whenever the models are updated. Can we opt to have users be able to set this / update this themselves if possible?

Reply:
The problem here is that, we rely on [`tiktoken-rs`](https://github.com/zurawiki/tiktoken-rs) to calculate our remaining tokens. If that package itself hasn't been updated to support the latest models, then we might incorrectly calculate what is left. We are sort of blocked on it.

Reply:
It looks like this new model is now supported on tiktoken (at least on the `main` branch, not sure if they have released a new version yet, but we could use https://github.com/zurawiki/tiktoken-rs/commit/f84907c7c77af25972027009837b16c75585de74.